jobs/kubelet/templates/bin/kubelet_ctl.erb

```diff
@@ -81,15 +84,15 @@ start_kubelet() {
     --docker-endpoint="${DOCKER_SOCKET}" \
     --docker="${DOCKER_SOCKET}" \
     --fail-swap-on=false \
+    --feature-gates=RotateKubeletServerCertificate=true \
     --hostname-override=${hostname_override} \
     --keep-terminated-pod-volumes=false \
     --kubeconfig=/var/vcap/jobs/kubelet/config/kubeconfig \
     --network-plugin=cni \
     --node-labels=<%= labels %>"$zone_label" \
     --read-only-port 0 \
+    --rotate-certificates \
```

Comment:
Once again, this flag comes with: `<Warning: Beta feature> Auto rotate the kubelet client certificates by requesting new certificates from the kube-apiserver when the certificate expiration approaches.`.  Are we confident in this features stability?